src/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs

```diff
@@ -739,27 +720,27 @@ private void SetGenParamCustomAttributeNoLock(CustAttr ca)
         #endregion
 
         #region Object Overrides
-        public override string ToString()
+        public override string? ToString()
         {
             return TypeNameBuilder.ToString(this, TypeNameBuilder.Format.ToString);
         }
 
         #endregion
 
         #region MemberInfo Overrides
-        public override Type DeclaringType
+        public override Type? DeclaringType
         {
             get { return m_DeclaringType; }
         }
 
-        public override Type ReflectedType
+        public override Type? ReflectedType
         {
             // Return the class that was used to obtain this field.
 
             get { return m_DeclaringType; }
         }
 
-        public override string Name
+        public override string? Name
```

Comment:
Can this ever return null?

Reply:
m_strName annotated as nullable ...

Reply:
https://github.com/dotnet/coreclr/blob/NullableFeature/src/System.Private.CoreLib/src/System/Reflection/Emit/TypeBuilder.cs#L437 doesn't initialize it

Reply:
this is fixed already - this one usage which doesn't initialize it is internal and doesn't touch Name so cannot NRE